recipes/r-scevan/meta.yaml

```diff
@@ -0,0 +1,50 @@
+package:
+  name: r-scevan
+  version: 1.0.1
+
+source:
+  url: https://github.com/AntonioDeFalco/SCEVAN/archive/refs/tags/v1.0.1.tar.gz  
+  sha256: 7558d31282c36418db3960b3edb9cb9815db14b0a48d8882287e11f752dfeab1       
+
+build:
+  number: 0
+  noarch: generic  # Specify that the package is noarch
+  run_exports: '{{ pin_compatible("r-scevan", max_pin="x.x") }}'
+  rpaths:
+    - lib/R/lib
+    - lib
+
+requirements:
+  host:
+    - r-base
+    - r-remotes
+    - r-devtools
+    - r-rcpp
+  run:
+    - r-base
+    - r-devtools
+    - r-paralleldist
+    - r-pheatmap
+    - r-forcats
+    - r-dplyr
+    - bioconductor-fgsea
+    - r-cluster
+    - r-ggplot2
+    - r-Rtsne
+    - bioconductor-scran
+    - r-ape
+    - bioconductor-ggtree
+    - r-tidytree
+    - r-ggrepel
+    - r-optparse
```

Comment:
Ok, the issue I think is that all of this are missing in host AFAIK, that is why is trying to install again all these dependencies.

Reply:
putting these in `host:` means that when it reaches the R CMD INSTALL --build for SCEVAN, then it won't try again to install them. Sorry, I hadn't paid attention to `run` and `host` sections. They also need to be present in run of course.

Reply:
fixed now